src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/Security/customers.html.twig

```diff
@@ -0,0 +1,55 @@
+{#**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ *#}
+
+{% extends '@PrestaShop/Admin/layout.html.twig' %}
+
+{% block content %}
+  {% block security_list_panel %}
+    <div class="row">
+      <div class="col">
+        {% include '@PrestaShop/Admin/Common/Grid/grid_panel.html.twig' with {'grid': grid} %}
+      </div>
+    </div>
+  {% endblock %}
+
+  {% block security_clear_customers_sessions %}
+    {% include '@PrestaShop/Admin/Configure/AdvancedParameters/Security/clear_form.html.twig' with {'route': 'admin_security_sessions_customer_clear'} %}
+  {% endblock %}
+{% endblock %}
+
+{% block javascripts %}
+  {{ parent() }}
+
+  <script src="{{ asset('themes/new-theme/public/security.bundle.js') }}"></script>
+  <script src="{{ asset('themes/default/js/bundle/pagination.js') }}"></script>
+{% endblock %}
+
+{% set js_translatable = {
+  "ModalTitle": 'Delete'|trans({}, 'Admin.Notifications.Warning'),
+  "ModalMessage": 'Are you sure you want to delete outdated sessions?'|trans({}, 'Admin.Notifications.Warning'),
```

Comment:
I think it's something I forget to remove because it previously was a modal :sweat:

Reply:
The most "dangerous" action would be to log out an old session, so it is not dangerous, but it can apply a massive data loss if applied, so the user needs to be sure but nothing dramatic